packages/jest-circus/src/__tests__/base_test-test.js

```diff
@@ -4,6 +4,8 @@
  * This source code is licensed under the BSD-style license found in the
  * LICENSE file in the root directory of this source tree. An additional grant
  * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow strict-local
```

Comment:
we used to have a lint rule that required flow in files, do we not have it any more?

Reply:
I think it was Danger.js responsible for that